lazy-imports-import.html

```diff
@@ -9,37 +9,71 @@
 -->
 <script>
   (function() {
-    var __loadedImports = {};
+    'use strict';
 
-    Polymer.__importLazyGroup = function(callingElement, group, fromElement) {
-      if (!fromElement) {
-        fromElement = Polymer.DomModule.import(callingElement.localName);
-      }
+    var __importPromises = {};
```

Comment:
Map is supported everywhere Polymer is, let's use that.

Reply:
@justinfagnani Does that include Polymer 1.x? This is the core implementation for both the behavior and mixin. Previously have excluded any ES6 functionality and enforced through linter

Reply:
IE11 has basic Map support, so it's ok

Reply:
Awesome! Thanks for the tip! Will get these changes made!